pkgs/tools/virtualization/distrobuilder/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, pkgconfig, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn
+, makeWrapper, coreutils, gnupg, gnutar, squashfsTools}:
+
+buildGoPackage rec {
+  name = "distrobuilder-unstable-${version}";
```

Comment:
I would just call it "distrobuilder-${version}" here. Having the date already shows it's not a stable release.